TeaDiary/Scripts/App/displayTeaList.js

```diff
@@ -0,0 +1,54 @@
+﻿function TeaListDisplayer(container) {
+    var self = this;
+    self.container = $(container);
+    function clearContainer() {
+        self.container.html("");
+    }
+    function setContainerLoading(loading) {
+        if (loading) {
+            self.container.hide();
+        } else {
+            self.container.show();
+        }
+    }
+    function buildTeaListHeader(tea) {
+        if (tea != null && tea.length > 0) {
+            return "<h2>Your tea list:</h2>";
+        } else {
+            return "<h2>No teas were found :(</h2>";
+        }
+    }
+    function getTeaSerialized(tea) {
```

Comment:
It's 'formatted', not 'serialized'. Also, `formatTea` reads a bit better.
If this function returned a string with a binary representation of the tea, then it would be 'serialize'